app/src/main/java/nu/yona/app/api/manager/dao/AuthenticateDAO.java

```diff
@@ -103,4 +127,32 @@ public User getUser()
 		}
 		return null;
 	}
+
+	public JSONObject getStoredUserObjectFromDB()
```

Comment:
This is overly complex. You can just annotate the ``User`` class with ``@JsonIgnoreProperties(ignoreUnknown = true)``. Then the property ``version`` won't be initialized if it's not in the database. No exceptions will occur.

Next time you need to create a method like this, be sure to prevent the code duplication. The only difference between this method and the next one is this:
* ``return new JSONObject(new String(bytes));``
* ``return serializer.deserialize(bytes, User.class);``

That can be implemented with no duplication at all.

Reply:
`@JsonIgnoreProperties(ignoreUnknown = true)`
This really makes it easy. I made it complex. I should have thought in this direction. Thanks.